models/issue_tracked_time.go

```diff
@@ -131,3 +186,114 @@ func TotalTimes(options FindTrackedTimesOptions) (map[*User]string, error) {
 	}
 	return totalTimes, nil
 }
+
+// DeleteIssueUserTimes deletes times for issue
+func DeleteIssueUserTimes(issue *Issue, user *User) error {
+	sess := x.NewSession()
+	defer sess.Close()
+	if err := sess.Begin(); err != nil {
+		return err
+	}
+
+	opts := FindTrackedTimesOptions{
+		IssueID: issue.ID,
+		UserID:  user.ID,
+	}
+
+	removedTime, err := GetTrackedSeconds(opts)
```

Comment:
The table needs to be locked for update before getting this result as whilst you're getting this number another update could come in and you'll miss it.